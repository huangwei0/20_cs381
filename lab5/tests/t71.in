findall(Z, ancestor(Z, 'Prince Henry of Wales'), Y), 
print(Y), 
print('\n'), 
halt.
