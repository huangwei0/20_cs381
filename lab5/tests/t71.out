[Charles: Prince of Wales,Lady Diana Spencer,King George III,King George III,Charlotte of Mecklenburg-Strelitz,Charlotte of Mecklenburg-Strelitz,Edward: Duke of Kent,Edward: Duke of Kent,Princess Victoria of Saxe-Coburg-Saalfeld,Princess Victoria of Saxe-Coburg-Saalfeld,Franz: Duke of Saxe-Coburg-Saalfeld,Franz: Duke of Saxe-Coburg-Saalfeld,Augusta Reuss of Ebersdorf und Lobenstein,Augusta Reuss of Ebersdorf und Lobenstein,Franz: Duke of Saxe-Coburg-Saalfeld,Franz: Duke of Saxe-Coburg-Saalfeld,Augusta Reuss of Ebersdorf und Lobenstein,Augusta Reuss of Ebersdorf und Lobenstein,Ernest I: Duke of Saxe-Coburg and Gotha,Ernest I: Duke of Saxe-Coburg and Gotha,Princess Louise of Saxe-Gotha-Altenburg,Princess Louise of Saxe-Gotha-Altenburg,Prince Albert,Queen Victoria,Prince Albert,Queen Victoria,Princess Alice,Louis IV: Grand Duke of Hesse,King Edward VII,Princess Alexandra of Denmark,King George V,Mary: Princess of Teck,Prince Louis of Battenberg,Princess Victoria of Hesse,Andrew: Prince of Greece,Alice: Princess of Baltenberg,King George VI,Lady Elizabeth Bowes-Lyon,Prince Phillip: Duke of Edinburgh,Queen Elizabeth II]
